en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/M/Morschecks_Hand.md

```diff
@@ -1,6 +1,27 @@
 #### Morscheck's Hand
 <!-- previously "Arcane Hand" -->
-<!-- TODO Check and tag this spell -->
+<!-- markdownlint-disable link-image-reference-definitions -->
+[_metadata_:spell_name]:- "Morscheck's Hand"
+[_metadata_:spell_original_name]:- "Arcane Hand"
+[_metadata_:spell_level]:- "5"
+[_metadata_:spell_school]:- "evocation"
+[_metadata_:ritual]:- "false"
+[_metadata_:casting_time_amount]:- "1"
+[_metadata_:casting_time_unit]:- "action"
+[_metadata_:range]:- "120 feet"
+[_metadata_:target]:- "an unoccupied space you can pinpoint"
+[_metadata_:components_verbal]:- "true"
+[_metadata_:components_somatic]:- "true"
+[_metadata_:components_material]:- "true"
+[_metadata_:components_material_description]:- "an eggshell and a snakeskin glove"
+[_metadata_:duration]:- "1 minute"
+[_metadata_:concentration]:- "true"
+[_metadata_:saving_throw]:- "Special"
+[_metadata_:saving_throw_success]:- "special"
+[_metadata_:damage_formula]:- "varies"
+[_metadata_:damage_type]:- "varies"
+[_metadata_:compared_to_wotc_srd_5.1]:- "mechanics_different_wording_different"
```

Comment:
The WotC version states:
> The hand [...] moves at your command, mimicking the movements of your own hand.

We don't have that, and it could be considered a limitation since you need to have one hand free to do whatever the spectral hand should do throughout the whole time it exists. With this version you can use your hands for other things once the spell has been cast.

We could add that back in, if we wanted to. What do you think?